api/config/cdr_config_prod.json

```diff
@@ -13,7 +13,7 @@
       "accessTierId": 2,
       "shortName": "controlled",
       "displayName": "Controlled Tier",
-      "servicePerimeter": "terra_prod_aou_prod_2",
+      "servicePerimeter": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_prod_2",
```

Comment:
FYI: I will rebase before merging, but I want this PR to be based on the current release tag for launch purposes; this change needs to be included